src/Kaleidoscope.h

```diff
@@ -36,29 +40,7 @@ extern HARDWARE_IMPLEMENTATION KeyboardHardware;
 const uint8_t KEYMAP_SIZE
 __attribute__((deprecated("Kaleidoscope.setup() does not require KEYMAP_SIZE anymore."))) = 0;
 
-class Kaleidoscope_;
-
-class KaleidoscopePlugin {
```

Comment:
Class `KaleidoscopePlugin` now lives in header `plugin.h` it was renamed to `kaleidoscope::Plugin`, thereby keeping the original name as deprecated type.